operators/src/util/raster_stream_to_png.rs

```diff
@@ -90,6 +90,8 @@ pub fn default_colorizer_gradient<T: Pixel>() -> Result<Colorizer> {
         ],
         RgbaColor::transparent(),
         RgbaColor::pink(),
+        None,
+        None,
```

Comment:
I would say… black for under and white for over instead of pink.

Reply:
There are a few more places like this. I hope I've caught all of them now.